### ISSUE TITLE
Design Panel: Alt text not readable when image can't load

### ISSUE THREAD

**Issue by samwhale** [swissspidy/web-stories-wp#9894]
## Bug Description

Alt text isn't readable when the image can't load in two places:
- image alt text is overflowing in the filter panel and the layers panel. Additionally, the contrast is too low to read the alt text in the filter panel
- alt text and image name get sort of hidden in the layer panel

## Expected Behaviour

1. Alt text should not overflow out of a container or be hidden.
2. Alt text should have a high enough contrast so users may read it.

## Steps to Reproduce

1. Use a very slow internet connection
2. Open the media 3p tab
3. Click on an image to add it in the canvas
4. Open the Filter panel
5. See the overflow and hidden text in the layer panel

## Screenshots

![image](https://user-images.githubusercontent.com/22185279/144822719-831104c0-663d-4e55-a8c4-166489d541fa.png)
![image](https://user-images.githubusercontent.com/22185279/144822812-c2fa5956-2837-4574-897c-bbafbe0590b4.png)

## Additional Context

- Plugin Version: 1.16.0
- WordPress Version: 5.8.2
- Operating System: macOSX Big Sur 11.5.2
- Browser: Chrome
